tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh

```diff
@@ -43,29 +45,42 @@ done
 
 shift $((OPTIND-1))
 
-if [ $# -ne 1 ]; then
-  >&2 echo "error: wrong argument (no argument or too many arguments)."
+if [ $# -eq 0 ]; then
+  >&2 echo "error: wrong argument (no model argument)."
   >&2 echo "For help, type $progname -h"
   exit 1
 fi
 
-modelfile=$(basename "$1")
+for modelpath in "$@"
+do
+  modelfile=$(basename "$modelpath")
 
-if [[ "$modelfile" != *.* ]]; then
-  >&2 echo "error: modelfile does not have extension."
-  >&2 echo "Please provide extension so that $progname can identify what type of model you use."
-  exit 1
-fi
+  if [[ "$modelfile" != *.* ]]; then
+    >&2 echo "error: modelfile does not have extension."
+    >&2 echo "Please provide extension so that $progname can identify what type of model you use."
+    exit 1
+  fi
 
-if [ ! -e $1 ]; then
-  >&2 echo "error: "$1" does not exist."
-  exit 1
-fi
+  if [ ! -e $modelpath ]; then
+    >&2 echo "error: "$modelpath" does not exist."
+    exit 1
+  fi
+
+  extension=${modelfile##*.}
+
+  if [[ "$models_str" == "" ]]; then
+    models_str="\"$modelfile\""
+    types_str="\"$extension\""
+  else
+    models_str="$models_str, \"$modelfile\""
+    types_str="$types_str, \"$extension\""
+  fi
```

Comment:
(optional) If you prefer _if_-less version like me,
```suggestion

  models_str="$models_str$delim \"$modelfile\""
  types_str="$types_str$delim \"${modelfile##*.}\""
  delim=","
i
```
Also, you need `delim=""` before entering _for_-loop.